ts/parts/Pointer.ts

```diff
@@ -854,6 +843,39 @@ Highcharts.Pointer.prototype = {
         });
     },
 
+    /**
+     * Set inactive state to all series that are not currently hovered,
+     * or, if `inactiveOtherPoints` is set to true, set inactive state to
+     * all points within that series.
+     *
+     * @function Highcharts.Pointer#setInactiveSeries
+     *
+     * @private
+     *
+     * @param {Array<Highcharts.Point>} points
+     *        Currently hovered points
+     *
+     */
+    setInactiveSeries: function (
```

Comment:
Same concern with prototype as for `Highcharts.Series#setAllPointsState`

Reply:
The same as `Highcharts.Series#setAllPointsState` - different files, affection for all series with `inactiveOtherPoints=true`.